tests/xharness/XmlResultParser.cs

```diff
@@ -6,15 +6,16 @@
 using System.Xml.Linq;
 
 namespace xharness {
-	public static class XmlResultParser {
 
-		public enum Jargon {
-			TouchUnit,
-			NUnitV2,
-			NUnitV3,
-			xUnit,
-			Missing,
-		}
+	public enum Jargon {
```

Comment:
My original comment (on the PR this `enum` was introduced) mentioned I disliked internal types ;-) because they are even harder to name than other types.

If you unnest the `enum` then it becomes less clear what the usage of the type is... (less self-documenting), e.g. the argument you're adding is `xml-jargon`. Maybe `Xml[Result]Jargon`, the prefix will help for intellisense.

Reply:
Funny thing is, I was thinking about your comment when I was doing the PR... he already mentioned it hehe